gym/lib/assets/wrap_xcodebuild/xcbuild-safe.sh

```diff
@@ -41,6 +44,22 @@ if [[ $? -eq 0 ]]; then
   unset BUNDLE_GEMFILE
 fi
 
+if command_exists rbenv; then
+  # Cause rbenv to use system ruby. Lasts only for the scope of this
+  # session which will normally just be this script.
+  rbenv shell system
+
+  # Unset environment variables that could break xcodebuild
+  unset RUBYLIB
+  unset RUBYOPT
+  uset _ORIGINAL_GEM_PATH
```

Comment:
Eep! @otaran @lacostej thanks for catching this, bad cludge from local test setup to my forked repo to submit the PR.